### PR TITLE
Document

### PR DESCRIPTION
**Pull request by Ludy87**
# Description

Corrected typographical errors across various documentation files, enhancing the clarity and professionalism of our guides.
Corrected "isnt" to "isn't" in HowToAddNewLanguage.md, ensuring grammatical accuracy.
Addressed an incorrect verb form, "doesnt" to "doesn't", in HowToUseOCR.md, improving readability.
Updated LocalRunGuide.md to correct "patern" to "pattern" and "arguements" to "arguments", refining the technical accuracy of the instructions.
In README.md, fixed a minor typo from "Sponser" to "Sponsor" and refined the wording for features under consideration, changing "mannual" to "manual".

## Checklist:

- [x] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ For API usage you must provide a header with 'X-API-Key' and the associated API
 - Redact text (Via UI not just automated way)
 - Add Forms
 - Multi page layout (Stich PDF pages together) support x rows y columns and custom page sizing
-- Fill forms mannual and automatic
+- Fill forms manual and automatic
```
Comment (sbplat) [inline README.md]: I think it should be
> Fill forms manually or automatically

File: LocalRunGuide.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ sudo systemctl restart stirlingpdf.service
 
 Remember to set the necessary environment variables before running the project if you want to customize the application the list can be seen in the main readme.
 
-You can do this in the terminal by using the `export` command or -D arguements to java -jar command:
+You can do this in the terminal by using the `export` command or -D arguments to java -jar command:
```
Comment (sbplat) [inline LocalRunGuide.md]: argument

File: README.md (hunk 1 above)
Comment (Ludy87) [reply README.md]: done

File: LocalRunGuide.md (hunk 2 above)
Comment (Ludy87) [reply LocalRunGuide.md]: done

**Review by sbplat — APPROVED** [review]
LGTM

**Pull request merged**